test/integration/host_js_test.rb

```diff
@@ -518,39 +518,6 @@ class HostJSTest < IntegrationTestWithJavascript
       assert page.has_no_selector?("#inherited_parameters #name_#{hostgroup1.group_parameters.first.name}")
       assert page.has_selector?("#inherited_parameters #name_#{hostgroup2.group_parameters.first.name}")
     end
-
-    # At this point in time this only tests the UI. Since Fog mocks all requests the Hard drive change is ignored as it
-    # will always return the mocked values. Not a total waste however as the UI is still tested with this.
-    test 'enables changing vm disk size' do
-      Fog.mock!
-
-      organization = Organization.find_by_name('Organization 1')
-      location = Location.find_by_name('Location 1')
-      uuid = '5032c8a5-9c5e-ba7a-3804-832a03e16381'
-      compute_resource = FactoryBot.create(:compute_resource, :vmware, organizations: [organization], locations: [location], uuid: 'Solutions')
-      compute_resource = ComputeResource.find_by_id(compute_resource.id)
-
-      host = FactoryBot.create(:host, :managed, compute_resource: compute_resource,
-                                                uuid: uuid,
-                                                location: location,
-                                                organization: organization,
-                                                provision_method: 'image')
-
-      visit edit_host_path(host)
-
-      wait_for_ajax
-
-      click_link('Virtual Machine')
-      within('.text-vmware-size') do
-        # Unusual here that the volume size input field has no name or id.
-        first('input').set('10 GB')
-      end
-
-      click_on('Submit')
```

Comment:
If we want to test that the ui creates the correct input in the hidden field, would it make sense to assert it's content here and skip the submit part? Or do we already have a unit test for that in the react code?

Reply:
We have this: https://github.com/theforeman/foreman/blob/b7854ee39cf7fd8bac71b745f1dd1db0e056edbb/webpack/assets/javascripts/react_app/components/hosts/storage/vmware/StorageContainer.test.js#L17 it tests that the component writes the internal state to the hiddne field properly.